### PR TITLE
POST /organizations/:id/courses/

### PR DESCRIPTION
**Pull request by mrkev**
*untested*

### CODE REVIEW

**Comment by Jma353** [comment]
@meganle can you review this?

File: src/routers/PostCourseRouter.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  getPath(): string {
+    return '/organizations/:id/courses/';
+  }
+
+  async content(req: Request): Promise<{ node: APICourse }> {
+    const name = req.body.name
+    const term = req.body.term
+    const orgId = req.params.id
+    const adminId = /** TODO: GET THIS FROM SENT CREDENTIALS */ 0;
+    const subject = /** TODO: Do courses really need a subject field? Discuss. */ '';
```
[the remark this answers is not in the document]
Comment (mrkev) [reply src/routers/PostCourseRouter.js]: 👍

**Comment by mrkev** [comment]
Ugh I messed up the merge on this one, I'll try to fix it soon.

**Comment by mrkev** [comment]
It has a TODO, but that will be dealt with once we have user sessions.

**Pull request merged**